src/model/table.py

```diff
@@ -0,0 +1,26 @@
+import json
+import os
+from model.index import *
+
+
+class TableMetaData:
+    def __init__(self, table_map):
+        self.table_map = table_map
```

Comment:
TableMetadata has properties, those properties should be extracted from table_map. 
TableMetadata doesn't have a table_map but it uses it to initialize its properties